elasticsearch/json-node-info.yml

```diff
@@ -0,0 +1,34 @@
+type: json
+data:
+  url: /_nodes/_local/os?format=smile
+  server: ${ST_MONITOR_ES_NODE_HOSTPORT}
+  basicHttpAuthUsername: ${ST_MONITOR_ES_NODE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${ST_MONITOR_ES_NODE_BASICAUTH_PASSWORD}
+  smileFormat: true
+require:
+  - className: com.sematext.spm.client.es.EsVersionCheck
+    values: 5-*
+
+observation:
+  - name: node_os_stats
+    metricNamespace: es
+    path: $.nodes.${nodeId}.os
+
+    metric:
+      - name: os.cpus.allocated
```

Comment:
Since we have just `disk.io....`, maybe go with `cpu.allocated.count` (also change to singular)?

Reply:
Just pushed a commit with this change and I've also updated the default chart.